### ISSUE TITLE
Account Information: Drupal User Roles

### ISSUE THREAD

**Issue by ercore** [EPSCoR/ercore8#5]
For the base install please add the following roles.
Note* can we import role permissions from D7 to D8? If not, I can attach a detailed list of what each roles permissions should be.

* Student (Undergrad, Graduate, Post Doc)
* Faculty or EPSCoR Researcher 
* Admin Staff 
* No longer with EPSCoR
* Guest
- Administrator

**Comment by tschet** [comment]
Do we still want to change the names? I think naming the after Senior roles is confusing.

**Comment by ercore** [comment]
I agree, lets not name after MSR. Let's go with the following titles, which is what you have already created, minus the text after the asterisk.

We need to talk about applying the actual permissions to each.

- ER Core Site Manager *(Previously Admin Staff)
- ER Core Edit Content *(previously Student)
- ER Core View Content *Only (Evaluator or Other)
- ER Core No Privileges *(Participants who are no longer on the project)
- Administrator
- Guest
